apps/dav/lib/CalDAV/CalDavBackend.php

```diff
@@ -2382,26 +2383,40 @@ protected function addChange($calendarId, $objectUri, $operation, $calendarType
 	 */
 	public function getDenormalizedData($calendarData) {
 		$vObject = Reader::read($calendarData);
+		$vEvents = [];
 		$componentType = null;
 		$component = null;
 		$firstOccurrence = null;
 		$lastOccurrence = null;
 		$uid = null;
 		$classification = self::CLASSIFICATION_PUBLIC;
+		$hasDTSTART = false;
 		foreach ($vObject->getComponents() as $component) {
-			if ($component->name !== 'VTIMEZONE') {
-				$componentType = $component->name;
-				$uid = (string)$component->UID;
-				break;
+			if ($component->name!=='VTIMEZONE') {
```

Comment:
```suggestion
			if ($component->name !== 'VTIMEZONE') {
```

cc @ChristophWurst seems there is a rule for php-cs-fixer missing.